reposilite-backend/src/main/kotlin/com/reposilite/token/AccessTokenFacade.kt

```diff
@@ -17,24 +17,36 @@ package com.reposilite.token
 
 import com.reposilite.token.AccessTokenSecurityProvider.B_CRYPT_TOKENS_ENCODER
 import com.reposilite.token.AccessTokenSecurityProvider.generateSecret
-import com.reposilite.token.api.AccessToken
-import com.reposilite.token.api.AccessTokenPermission
-import com.reposilite.token.api.AccessTokenType
+import com.reposilite.token.api.*
```

Comment:
We shouldn't use wildcard imports at all, especially in Kotlin to avoid strange issues in some specific cases and keep all references clean :)

Reply:
Alright, believe Intellij automatically did this so I'll quickly revert it